From 74a1b243872a3882c5334ce1e51edd4630a39cd9 Mon Sep 17 00:00:00 2001
From: Wang Panzhenzhuan <randy.wang@rock-chips.com>
Date: Wed, 11 Dec 2019 11:23:40 +0800
Subject: [PATCH] media: i2c: ov5695: fix mistake

Commit b7526773a3c5 ("media: i2c: add enum_frame_interval
function for iq tool 2.2 and hal3") add enum_frame_interval
function, but forget to use in struct v4l2_subdev_pad_ops, fix it.

Fixes: b7526773a3c5 ("media: i2c: add enum_frame_interval function for iq tool 2.2 and hal3")
Change-Id: I8383a7a409ca5f2fa3a8f8d98c38b94062c2bb36
Signed-off-by: Wang Panzhenzhuan <randy.wang@rock-chips.com>
---
 drivers/media/i2c/ov5695.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/drivers/media/i2c/ov5695.c b/drivers/media/i2c/ov5695.c
index b10868599723..79eaa98447c7 100644
--- a/drivers/media/i2c/ov5695.c
+++ b/drivers/media/i2c/ov5695.c
@@ -1163,6 +1163,7 @@ static const struct v4l2_subdev_video_ops ov5695_video_ops = {
 static const struct v4l2_subdev_pad_ops ov5695_pad_ops = {
 	.enum_mbus_code = ov5695_enum_mbus_code,
 	.enum_frame_size = ov5695_enum_frame_sizes,
+	.enum_frame_interval = ov5695_enum_frame_interval,
 	.get_fmt = ov5695_get_fmt,
 	.set_fmt = ov5695_set_fmt,
 };
-- 
2.35.3

